### PR TITLE
Added the Documentation site and Description of each component

### PR DESCRIPTION
**Pull request by atharvabhanage02**
Added 

- Landing Page
- Documentation website
- Description for each component

Here is the link of the final website 
https://deploy-preview-9--suspicious-lalande-55f8fd.netlify.app/index.html

### CODE REVIEW

**Review by abhijeetekad — APPROVED** [review]

File: documentation/docs.css [hunk 1]
```diff
@@ -0,0 +1,189 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+body{
+    font-family: 'Montserrat', sans-serif;
+}
```
Comment (abhijeetekad) [inline documentation/docs.css]: Suggestion: Add scroll-behavior: smooth;

Comment (atharvabhanage02) [reply documentation/docs.css]: Done 
Thanks for the suggestion

File: documentation/docs.css [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+:root{
+    --component-library-name-font : 2.5rem;
+    --cl-color : #4d4dff;
+    --sidebar-background-color:white;
+    --sidebar-text-color:white;
+}
+*{
+    margin: 0;
+    padding: 0;
```
Comment (MiheerTamkhane) [inline documentation/docs.css:14]: i think box-sizing : border-box. is good to have.

**Review by MiheerTamkhane — APPROVED** [review]
Looks good to merge. One thing I would suggest you are using too much specific values for spacing like 4.37rem, 0.625rem try to avoid and use as simple as possible.

**Comment by atharvabhanage02** [comment]
> Looks good to merge. One thing I would suggest you are using too much specific values for spacing like 4.37rem, 0.625rem try to avoid and use as simple as possible.

Thanks for pointing out that .
I'll avoid using too much specific values.

File: documentation/docs.css (hunk 2 above)
Comment (atharvabhanage02) [reply documentation/docs.css:14]: Yeah . I'll do that

**Pull request merged**